egs2/meld/asr1/run.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set="train"
+valid_set="valid"
+test_sets="valid test"
+
+asr_config=conf/tuning/train_asr_hubert_transformer_adam_specaug_meld.yaml
```

Comment:
Please use a soft link and make this as the `conf/train_asr.yaml` for consistency to other recipes.

Reply:
If I understand you right, do you suggest softlink a `conf/train_asr.yaml` to `conf/`, and replace line12 with `asr_config=conf/train_asr.yaml`, or just add the soft link and keep the tuning configs? Some recipes use tuning config in their `run.sh`.

Reply:
softlink a conf/train_asr.yaml to your actual config(conf/,) and replace line12 with asr_config=conf/train_asr.yaml